refactor/src/Cage.py

```diff
@@ -418,13 +426,36 @@ def do_mating_events(self):
                 df_list[fc].loc[df_list[fc].index.isin(dams),'mate_resistanceT1'] = \
                 df_list[fc].loc[df_list[fc].index.isin(sires),'resistanceT1'].values
         """
-        pass
 
-    def create_offspring(self):
+        # for now only consider a sensible number of lice that can mate
+        return min([10, self.lice_population["L5f"], self.lice_population["L5m"]])
```

Comment:
Why is 10 a sensible number?

Reply:
I think I did this for testing, or at least I planned to but ended up not doing anything with it anyway. I'll revert this.